### PR TITLE
TSK-1717: The JointPoint for our LoggingAspect now only looks for classes within pro.taskana package

### PR DESCRIPTION
**Pull request by mustaphazorgati**
https://sonarcloud.io/dashboard?branch=task%2F1717&id=mustaphazorgati_taskana

---
 
### For the submitter:
- [ ] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
- [x] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
- [x] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
- [x] I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [ ] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I put the ticket in review
- [ ] After integration of the pull request, I verified our [bluemix test environment](http://taskana.mybluemix.net/taskana) is not broken

### Verified by the reviewer:
- [ ] Commit message format → TSK-XXX: Your commit message.
- [ ] Submitter's update to [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) is sufficient
- [ ] SonarCloud analysis meets our standards
- [ ] Update of the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana) reflects changes
- [ ] PR fulfills the ticket
- [ ] Edge cases and unwanted side effects are tested
- [ ] Readability

### CODE REVIEW

File: common/taskana-common-logging/src/test/java/outside/of/pro/staskana/OutsideOfProTaskanaPackageLoggingTestClass.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package outside.of.pro.staskana;
```
Comment (benjamineckstein) [inline common/taskana-common-logging/src/test/java/outside/of/pro/staskana/OutsideOfProTaskanaPackageLoggingTestClass.java:1]: staskana

**Review by benjamineckstein — COMMENTED** [review]
👍

**Review by benjamineckstein — APPROVED** [review]
